build/tooling/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.15.4
```

Comment:
Can we go with `1.15-alpine` here to reduce the size?

Reply:
Done.  

I went with the regular image to start because of my past painful experiences with `apk`, and because this image will never be run in parallel on a machine where size might matter.